s3-uploads.php

```diff
@@ -21,6 +21,13 @@ function s3_uploads_init() {
 		return;
 	}
 
+	// Make sure the plugin is enabled when autoenable is on, or in a CLI system (so the migrate command works)
+	if ( ( ! defined( 'WP_CLI') || ! WP_CLI ) &&                                      // If CLI is enabled, skip our other checks
```

Comment:
In the case where someone has `enabled` off, and they do something in WP CLI, this is going to have the unintended affect of enabling s3-uploads when it shouldn't be. 

A better approach I think would be to do 

```PHP
$instance = S3_Uploads::get_instance();
$instance->register_stream_wrapper();
```

In the wp-cli s3-uploads command in the case of it being disabled.